server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java

```diff
@@ -0,0 +1,41 @@
+package org.infinispan.server.resp.commands.hash;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.multimap.impl.EmbeddedMultimapPairCache;
+import org.infinispan.server.resp.Consumers;
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespErrorUtil;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.Resp3Command;
+
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * `<code>HGET key field</code>` command.
+ * </p>
+ * Find the value associated with field in the hash stored at key.
+ *
+ * @since 15.0
+ * @see <a href="https://redis.io/commands/hget/">Redis Documentation</a>
+ * @author José Bolina
+ */
+public class HGET extends RespCommand implements Resp3Command {
+
+   public HGET() {
+      super(3, 1, 1, 1);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, ChannelHandlerContext ctx, List<byte[]> arguments) {
+      if (arguments.size() != 2) {
```

Comment:
I did the arity check PR. Please merge that before this one

Reply:
Taking a look.